holder.go

```diff
@@ -70,7 +70,7 @@ func NewHolder() *Holder {
 }
 
 // Open initializes the root data directory for the holder.
-func (h *Holder) Open() error {
+func (h *Holder) Open(logOutput io.Writer) error {
```

Comment:
That was my first choice, but I backed off, because currently it would have to go through NewCommand->NewServer->NewHolder.